src/components/MediaObject/__snapshots__/MediaObject.test.tsx.snap

```diff
@@ -173,7 +165,6 @@ exports[`<MediaObject /> with title content matches its snapshot 1`] = `
     gutterSize="medium"
   >
     <FixedGridColumn
-      className="y-media-object__size-medium--image"
```

Comment:
Is there anything we can delete from the CSS file now that these class names are gone?

Reply:
yes there's related CSS. but we need to hear from @adrianmg as the original 33px height was deliberate